pipeline/scripts/average_rating.py

```diff
@@ -0,0 +1,37 @@
+import sys
+
+sys.path.insert(0, "/workspace/butterfly/backend/utils/")
+
+
+def calculate_analytics(matches):
```

Comment:
Nice job splitting this function out! That will make it easier to test.

Optional: You can also add a type hint to tell the reader what types the inputs and outputs should be. For example:

```python
from typing import Dict, List

from pipeline.types import UserId

def get_average_user_ratings(ratings: List[Dict]) -> Dict[UserId, float]:
  ...
  return averages
```

Also optional: consider renaming the function to describe what it does, like `get_average_user_ratings()`.